core/src/main/scripts/importer/validateData.py

```diff
@@ -817,9 +815,10 @@ def checkGeneIdentification(self, gene_symbol=None, entrez_id=None):
 
         # check whether at least one is present
         if entrez_id is None and gene_symbol is None:
-            self.logger.error(
-                'No Entrez gene id or gene symbol provided for gene.',
-                extra={'line_number': self.line_number})
+            if profile_type != 'SV':
```

Comment:
is this based on a file or db? in db datatype is SV but genetic_alteration_type is STRUCTURAL_VARIANT

Reply:
This is just a keyword passed in https://github.com/cBioPortal/cbioportal/blob/0c9bf854bc700e5dc0e8bc6c391a8d6e9c373755/core/src/main/scripts/importer/validateData.py#L3097 for the file type. Could have used the meta dictionary but the unit tests doesn't recognize so.